.github/workflows/release.yaml

```
@@ -44,7 +44,7 @@ jobs:
     needs:
       - lib-check
       - ci-tests
-    uses: canonical/data-platform-workflows/.github/workflows/release_charm_edge.yaml@v32.0.0
+    uses: canonical/data-platform-workflows/.github/workflows/release_charm_edge.yaml@v31.1.1
     with:
       track: '8.0'
```

Reverting PR https://github.com/canonical/mysql-k8s-operator/pull/603

```suggestion
      channel: 8.0/edge
```